src/CoreText/CTFontManager.cs

```diff
@@ -156,7 +215,9 @@ static NSArray EnsureNonNullArray (object [] items, string name)
 			}
 		}
 
+#if !NET
 		[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
+#endif
```

Comment:
Missing 10.15 for NET since the min is 10.14.

Reply:
@mandel-macaque that's fine, the dotnet attribute can't be applied to delegates - which is half true (the delegate can be used whatever the OS version) and half false (you can keep looking for an available API that can consume it)